src/NzbDrone.Core.Test/NotificationTests/Xbmc/Http/GetArtistPathFixture.cs

```diff
@@ -48,47 +48,47 @@ public void Setup()
         }
 
         [Test]
-        public void should_get_series_path()
+        public void should_get_artist_path()
         {
-            const string queryResult = @"<xml><record><field>smb://xbmc:xbmc@HOMESERVER/TV/30 Rock/</field></record></xml>";
```

Comment:
Should we use artist names rather than TV show names in these tests and the ones after?

Reply:
Probably a good idea